cfgov/v1/models/molecules.py

```diff
@@ -63,12 +64,17 @@ class Hero(blocks.StructBlock):
     heading = blocks.CharBlock(required=False)
     body = blocks.RichTextBlock(required=False)
 
-    image = atoms.ImageBasic()
+    background_image = ImageChooserBlock(required=False,
+                                         help_text='An image object containing the URL of the image to be placed behind the hero.')
+
+    image = atoms.ImageBasic(required=False)
```

Comment:
this is a remnant from when we had not overridden the Wagtail Image Model and included alt. `ImageBasic` needs to be deleted. but i'd rather that done in a separate Issue.